### ISSUE TITLE
[BUG] Creating availability fails due to quota error despite available storage

### ISSUE THREAD

**Issue by 2-towns** [codex-storage/nim-codex#919]
**Describe the bug**
I created approximately 16 availabilities, consuming 7.1 GB of my 8 GB quota. When I attempt to create a new availability of 0.85 GB, the API returns a quota error.

**To Reproduce**
In my case I have multiple availabilities.  

The API response I am receiving when calling the storage node api is: 
```json
{
    "totalBlocks": 9276,
    "quotaMaxBytes": 8589934592,
    "quotaUsedBytes": 607193167,
    "quotaReservedBytes": 7623566944
} 
```

8589934592 - 7623566944 = 966367648 free bytes. 

My availability request data are: 
```json
{
    "totalSize": "912680550",
    "duration": "86400",
    "minPrice": "0",
    "maxCollateral": "0"
}
```
**Expected behavior**
912680550 < 966367648 so the request should work. 

**Screenshots**
![Screenshot from 2024-09-25 12-22-19](https://github.com/user-attachments/assets/b2c4ae37-bb57-45e7-80cf-d26d30c37397)
![Screenshot from 2024-09-25 12-22-35](https://github.com/user-attachments/assets/3b06447f-8d71-439f-8b98-5c5e10134ce0)
![Screenshot from 2024-09-25 12-22-47](https://github.com/user-attachments/assets/6e903699-6a6d-4706-a6bf-09aadb74e73b)

**Environment:**
- OS: 
- Version: 
- Tag/Commit: 
- Arch: amd64

**Additional context**

**Comment by AuHau** [comment]
This is correct behaviour because the limit is not based only on the `quotaReservedBytes` but the "whole usage" which is `quotaReservedBytes` and `quotaUsedBytes`, which together must not exceed the `quotaMaxBytes`. 

So in your case `7623566944 + 607193167 = 8230760111` ==> `8589934592 - 8230760111 = 359 174 481` you have only 359MB available to use ;-)

The question here is, do you think docs should be improved in some way in order for users to have better understanding and grasps of this concept?

**Comment by 2-towns** [comment]
Thanks for the explanation!

I believe we should update the documentation, as I initially thought that `quotaReservedBytes` represented all data reserved by the availabilities, including `quotaUsedBytes`.

The description for `quotaUsedBytes` in OpenAPI is 'Amount of storage space currently in use.' I understand this to mean the amount of space currently occupied by files, correct?

**Comment by AuHau** [comment]
Yeah "used" bytes are those that are physically stored in the repo - eq. used by some file. 
"Reserved" bytes are reserved by some availability for future use when storage request are matched and slots filled with the availability.

Does that help? Could you then please reword this part of documentation to make it more clear?

**Comment by 2-towns** [comment]
> Does that help? 

Yeah it' clear to me. 

> Could you then please reword this part of documentation to make it more clear?

Yes, I will take care of that.
